config/src/main/kotlin/org/springframework/security/config/annotation/web/AbstractRequestMatcherDsl.kt

```diff
@@ -42,8 +44,19 @@ abstract class AbstractRequestMatcherDsl {
                                                   val httpMethod: HttpMethod? = null,
                                                   override val rule: String) : AuthorizationRule(rule)
 
+    protected data class MatcherAuthorizationManagerRule(val matcher: RequestMatcher,
```

Comment:
Please place this immediately after `MatcherAuthorizationRule`, as you did with `AuthorizationManagerRule`

Reply:
Done